<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rozgrywka</class>
 <widget class="QMainWindow" name="Rozgrywka">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EURObiznes</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="gvPlansza">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>10</y>
      <width>950</width>
      <height>950</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="gvPanelGracza">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>600</width>
      <height>300</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbKostki">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>810</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <activeoff>:/Zasoby/grafika/ikony_przyciskow/kostki.png</activeoff>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbNastRunda">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>870</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <activeoff>:/Zasoby/grafika/ikony_przyciskow/nastRunda.png</activeoff>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>45</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbKup">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>810</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <disabledoff>:/Zasoby/grafika/ikony_przyciskow/kup.png</disabledoff>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbWlasnosci">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>810</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <disabledoff>:/Zasoby/grafika/ikony_przyciskow/dom.png</disabledoff>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pbUwolnij">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>870</y>
      <width>110</width>
      <height>50</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Wyjdź z więzienia</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGra">
    <property name="title">
     <string>Gra</string>
    </property>
    <addaction name="actionZapisz"/>
    <addaction name="actionWyjscie_2"/>
   </widget>
   <addaction name="menuGra"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionZapisz">
   <property name="text">
    <string>Zapisz</string>
   </property>
  </action>
  <action name="actionZgry">
   <property name="text">
    <string>Zgry</string>
   </property>
  </action>
  <action name="actionWyjscie_2">
   <property name="text">
    <string>Wyjście</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
